diff --git setup.py setup.py
index a20eb58..9fc68ea 100644
--- setup.py
+++ setup.py
@@ -53,7 +53,7 @@ include_dirs = [
     os.path.dirname(get_python_inc()),
     get_python_inc()
 ]
-library_dirs = [get_config_var("LIBDIR")]
+library_dirs = [get_config_var("LIBDIR")] if os.name == "posix" else []
 sources = glob("src/*.pxd") + glob("src/*.pyx") + glob("src/*.cpp")
 libraries = ["boost_container"]
 extra_compile_args = []

